homeassistant/util/unit_conversion.py

```diff
@@ -29,6 +30,17 @@
 )
 from homeassistant.exceptions import HomeAssistantError
 
+# Area constants to square meters
+_CM2_TO_M2 = 1e-4  # 1 cm² = 0.0001 m²
+_MM2_TO_M2 = 1e-6  # 1 mm² = 0.000001 m²
+_KM2_TO_M2 = 1e6  # 1 km² = 1,000,000 m²
+_IN2_TO_M2 = 0.00064516  # 1 in² = 0.00064516 m²
+_FT2_TO_M2 = 0.092903  # 1 ft² = 0.092903 m²
+_YD2_TO_M2 = 0.836127  # 1 yd² = 0.836127 m²
+_MI2_TO_M2 = 2.59e6  # 1 mi² = 2,590,000 m²
+_ACRE_TO_M2 = 4046.86  # 1 acre = 4,046.86 m²
+_HECTARE_TO_M2 = 10000  # 1 hectare = 10,000 m²
```

Comment:
The constants for converting US customary units are aggressively rounded and just magic numbers, that's not OK
Do like this instead:
```py
_CM2_TO_M2 = _CM_TO_M**2  # 1 cm² = 0.0001 m²
...
_MILE2_TO_M2 = _MILE_TO_M**2  # 1 mi² = 2,590,000 m²
_ACRE_TO_M2 = 66 * unit_conversion._FOOT_TO_M * 660 * unit_conversion._FOOT_TO_M  # ...
_HECTARE_TO_M2 = 100 * 100  # 1 hectare = 10,000 m²
```